pkg/mycarehub/usecases/communities/communities.go

```diff
@@ -323,3 +327,27 @@ func (us *UseCasesCommunitiesImpl) RejectInvite(ctx context.Context, userID stri
 
 	return true, nil
 }
+
+// AcceptInvite accepts an invite into a community
+func (us *UseCasesCommunitiesImpl) AcceptInvite(ctx context.Context, userID string, channelID string) (bool, error) {
+	if userID == "" {
+		return false, fmt.Errorf("user id is empty")
```

Comment:
The error should be like ``user id cannot ...``
Same for channel error message...